bin/generate-docs

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# this script edits the sources allowing the autodoc to import them, runs the generation, then reverts the changes.
+
+format='html'
+if [ $# -eq 1 ] ; then
+  format=$1
+fi
+
+mv hmt_escrow/storage.py hmt_escrow/storage.py.backup
+mv hmt_escrow/eth_bridge.py hmt_escrow/eth_bridge.py.backup
+python sphinx-documentation/escape_code.py storage
+python sphinx-documentation/escape_code.py eth_bridge
+
```

Comment:
is this something we should do in code permanently?

Reply:
You mean if we should refractor the code so we don't need the escape_code script, or if the script still works if the code changes?

Reply:
Both, yes.  Seems odd to me that the code is tweaked temporarily to generate docs; does the generate function not like those lines?  Does the code break if its permanently changed in this way?

Reply:
As discussed above, I also think the script is a bad idea and the code should be fixed, as the files are not classes, and currently there is initialization code hanging around without being in a function. So it can't be imported without side-affect calls.

Reply:
That discussion being https://github.com/hCaptcha/hmt-escrow/issues/72#issuecomment-590786691 ?  

Gotcha; yeah I'd agree, probably scope for another pr

Reply:
Yes that sounds good. The places where they are used have to be then adjusted to instantiate or initialize these when needed. Where are they used?